prow/cmd/branchprotector/BUILD

```diff
@@ -1,3 +1,44 @@
+# Usage:
+#   bazel run :dev-job.create
+load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_bundle")
+load("@io_bazel_rules_docker//contrib:push-all.bzl", "docker_push")
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+k8s_object(
+    name = "dev-job",
+    cluster = "gke_k8s-prow_us-central1-f_prow",  # Run make :get-cluster-credentials
+    images = {
+        # TODO(fejta): ideally {STABLE_DOCKER_REPO}
+        # Needs to match the value in dev-job.yaml
```

Comment:
Yeah, without templating (and `rules_jsonnet` [doesn't support stamping](https://github.com/bazelbuild/rules_jsonnet/issues/29)), I don't have a good answer for this.

However, even if this is a fixed string, `image_chroot` is applied after the match, and controls where the image is *actually* pushed (and then the resolved digest is injected in place of this fixed key).

Reply:
Done